doc/mgr/telemetry.rst

```diff
@@ -54,6 +59,12 @@ You can look at what data is reported at any time with the command::
 
   ceph telemetry show
 
+To protect your privacy, device reports are generated separately, and data such
+as hostname and device serial number is anonymized. To see a preview of the
```

Comment:
The device telemetry is send to a different endpoint, storted in a different database, and does not associate the device data with a particular cluster.

Reply:
added;
added also a note about smartmontools version.